release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2154,6 +2154,9 @@ As a temporary workaround, the image registry should not be configured as privat
 
 [id="ocp-telco-ran-4-17-known-issues_{context}"]
 
+* When you run Cloud-native Network Functions (CNF) latency tests on an {product-title} cluster, the test can sometimes return results greater than 20 microseconds. This results in a test failure.
```

Comment:
Since we're potentially covering multiple tests here, we should make this more generic - how about:
When you run Cloud-native Network Functions (CNF) latency tests on an {product-title} cluster, the test can sometimes return results greater than the latency threshold for the test (e.g. 20 microseconds for cyclictest testing). This results in a test failure.